### ISSUE TITLE
Give Chromium access to xdg-download

### ISSUE THREAD

**Issue by thatLeaflet** [flathub/org.chromium.Chromium#362]
The Chromium flatpak by default has access to the entire home folder. On most Linux system, this works fine as most people have their download folders in their home folder.

However, some people, like me, have their XDG download folder on secondary drives. When attempting to download anything, the download operation silently fails (as in Chromium doesn't report any issues, but the file isn't anywhere in the filesystem).

This can be worked around with using Flatseal to grant Chromium access to xdg-download, but it would be better if it had this access by default.

**Comment by thatLeaflet** [comment]
Yes, although the xdg-download path is the most important. I haven't run into any issues with the other xdg paths because the file picker portal can handle those.

**Comment by AndreiSva** [comment]
To add to this, I think giving Chromium access to the whole home directory is a bit extreme and potentially bad for security.

**Comment by TrusterX** [comment]
i second that - i removed access to home.

I wouldn't add any other directory, as uploads are handled through the Portal anyway, so there is no need to add them, i guess. And if i have to test a local website, i just copy it in the download directory... or you create a designated folder for this use case and give accress to Chromium (for eg. ~/.local/www/)
